src/seedu/addressbook/common/Utils.java

```diff
@@ -13,6 +13,7 @@
      * Returns true if any of the given items are null.
      */
     public static boolean isAnyNull(Object... items) {
+        if (items == null) return true;
```

Comment:
Can you tell me what this line does and why you use it? :)

Reply:
To avoid an error if isAnyNull is called on just a single null.

Reply:
👍 You can close this PR now